packages/schematics/angular/migrations/update-6/index.ts

```diff
@@ -279,7 +279,7 @@ function extractProjectsConfig(config: CliConfig, tree: Tree): JsonObject {
             ...(isProduction && serviceWorker ? { serviceWorker: true } : {}),
             fileReplacements: [
               {
-                src: `${app.root}/${source}`,
+                source: `${app.root}/${source}`,
```

Comment:
The form I'd like to see is `{ replace: "some/file/path", with: "some/other/file" }` which to mean feels more natural. What do you think?

Reply:
I'm ok with that.